frontend/src/gql/graphql.ts

```diff
@@ -1212,6 +1229,27 @@ export type UserAgent = {
   version?: Maybe<Scalars['String']['output']>;
 };
 
+export type UserConnection = {
+  __typename?: 'UserConnection';
+  /** A list of edges. */
+  edges: Array<UserEdge>;
+  /** A list of nodes. */
+  nodes: Array<User>;
+  /** Information to aid in pagination. */
+  pageInfo: PageInfo;
+  /** Identifies the total count of items in the connection. */
+  totalCount: Scalars['Int']['output'];
+};
+
+/** An edge in a connection. */
+export type UserEdge = {
+  __typename?: 'UserEdge';
+  /** A cursor for use in pagination */
+  cursor: Scalars['String']['output'];
+  /** The item at the end of the edge */
+  node: User;
+};
```

Comment:
I don't understand what these are

Reply:
All the paginated lists are like this in the GraphQL API. This is following the 'Connection' convention from Relay: https://graphql.org/learn/pagination/#complete-connection-model

Basically it normalises how you do paginated lists, where each item (= edge) has a cursor, and you can paginate before and after a cursor